config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     logLevel: info
     enableKubeProbes: true
     healthPort: 8080
+    promPort: 9201
```

Comment:
it's 9201 here vs 9102 by default .. on purpose, or a mistake?

Reply:
Fixed, a mistake ... I changed all of them to 9090